src/screens/login/components/assets/PasswordView/htmlPasswordInjection.ts

```diff
@@ -35,7 +35,12 @@ const locale = 'fr'
  * @param {string} backgroundColor - The LoginScreen's background color (used for overlay and navigation bars)
  * @returns {Element} HTML of Password form to inject inside Webview
  */
-export const getHtml = (title, fqdn, instance, backgroundColor) => {
+export const getHtml = (
+  title: string,
+  fqdn: string,
+  instance: string,
+  backgroundColor: string
+): string => {
```

Comment:
la JSDoc dit ` * @returns {Element} HTML of Password form to inject inside Webview` et le TS dit string. Qui qui gagne ?

Reply:
Corrigé ici: https://github.com/cozy/cozy-flagship-app/compare/629d70cb1d5287583ea6504d10a49e7e54d33b83..f032b3f25aca1ff82737b3f5705ae5e4b32021eb

Techniquement c'est toujours TS qui gagne vu qu'il ne peut pas être faux (là où la JSDoc ne fait pas de vérif, du moins sur ce projet)